R/config-log.R

```diff
@@ -81,32 +81,33 @@ config_log_impl <- function(.mods) {
 
   check_model_object_list(.mods)
 
+  # Filter out unfinished models - this is specifically necessary for bootstrap
+  # runs, which have a config file _before_ submission.
+  mods_finished <- map_lgl(.mods, function(.m) model_is_finished(.m))
```

Comment:
Yeah `bbi_nonmem_model_status()` should probably just be `bbi_model_status()` or even `model_status()` (unsure if `bbr.bayes` stan models use `bbi`). I didnt change the name in https://github.com/metrumresearchgroup/bbr/pull/693 just because the name had been around forever, but agree it's a bit misleading. Can change in a future PR